lib/providers/cartobio.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const pool = require('../db.js')
+
+/**
+ * Populate third party informations with our very own operators data
+ *
+ * @param  {Array.<numeroBio>} operators An array of objects fetched from the Agence Bio Notifications portal
+ * @return {[type]}           The same array, merged with our own local data
+ */
+function populateWithMetadata (operators) {
+  const ids = operators.map(({ numeroBio }) => numeroBio)
+
+  return pool
+    .query(`SELECT numerobio, pacage FROM cartobio_operators WHERE numerobio IN (${ids.join(',')})`)
```

Comment:
C'est un peu nul, mais j'ai pas encore trouvé comment faire en sorte que la version paramétrique ne caste pas les items en `String`. Postgres crache une erreur car les items du `IN()` sont des chaines alors que la colonne est un entier.

```js
query('SELECT numerobio, pacage FROM cartobio_operators WHERE numerobio IN ($1)', [ids])
```

Cette variante me jette une erreur comme quoi le filtre n'existe pas :

```js
query('SELECT numerobio, pacage FROM cartobio_operators WHERE numerobio IN ($1::integer[])', [ids])
```